projects/aca-playwright-shared/src/api/nodes-api.ts

```diff
@@ -418,6 +418,26 @@ export class NodesApi {
     }
   }
 
+  async getFileVersionType(nodeId: string): Promise<string> {
```

Comment:
Do you think we need a separate function for that? `getNodeProperty` function already catches potential errors so I think instead of adding this function you can just directly use `const fileVersionType = await this.getNodeProperty(nodeId, 'cm:versionType');` in the test itself

Reply:
I change this code with getNodeProperty